tracee/tracee.bpf.c

```diff
@@ -495,15 +497,34 @@ static __always_inline int get_config(u32 key)
     return *config;
 }
 
+static __always_inline int uid_filter_matches()
+{
+	uid_t uid = (u32)bpf_get_current_uid_gid();
```

Comment:
Please use spaces instead of tabs (this what happens when we have both go and c in the same repo)